units/commit_queue.go

```diff
@@ -665,19 +665,13 @@ func addMergeTaskAndVariant(patchDoc *patch.Patch, project *model.Project, proje
 	}
 
 	// Merge task depends on all the tasks already in the patch
-	status := ""
```

Comment:
Can you explain why we're removing this?

Reply:
this makes the merge task not run if a normal task fails for pull requests, so it's consistent with CLI items now. The only reason it ran before was to send a status to github